presto-clickhouse/src/main/java/com/facebook/presto/plugin/clickhouse/ClickHouseMetadataConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.clickhouse;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+
+public class ClickHouseMetadataConfig
+{
+    private boolean allowDropTable;
+
+    public boolean isAllowDropTable()
+    {
+        return allowDropTable;
+    }
+
+    @Config("allow-drop-table")
```

Comment:
how about we merge `ClickHouseMetadataConfig` into `ClickhouseConfig`?
`clickhouse.allow-drop-table`